config/locales/jobs.yml

```diff
@@ -146,7 +146,9 @@ en:
     trust_type: Trust type
     expired_tag: expired
     listing_expired: This job post has expired.
-    listing_expired_on_html: "This job expired on <span class='govuk-!-font-weight-regular'>%{date}</span>"
+    listing_expired_on_html: >-
+      This job expired on <span class='govuk-!-font-weight-regular'>%{date} -
+      <a href=#similar-jobs class='govuk-link govuk-link--no-visited-state'>[see similar jobs]</a></span>
```

Comment:
Is there a reason we can't do the html in the slim file, rather than in the translation?

Reply:
We pass this into the Timeline component's heading slot (I think that's the correct term lol). I'm unsure of how I could render the link without providing the html for it in the translation that gets passed into the slot?

Reply:
See here for example of how to pass links to translations: https://github.com/DFE-Digital/teaching-vacancies/blob/1d78783c9b8ac6baa4a723109c706932efba7212/app/views/home/index.html.slim#L22